index.js

```diff
@@ -4,7 +4,7 @@ const path = require('path');
 const childProcess = require('child_process');
 const fs = require('fs');
 const isWsl = require('is-wsl');
-const isDocker = require('is-docker');
+const isDocker = require('is-docker')();
```

Comment:
The function should be called when used, not at package load time.

Reply:
I'll make amends. My reasoning was that since we're checking the environment it's literally running in, this check should remain stable throughout the module's lifetime. I can see how this might cause problems for testing and such though (although the same would be true for `isWsl`).

Reply:
Not just that, but it's an expensive call and you don't want to hurt startup performance.